src/helpers.js

```diff
@@ -293,3 +300,10 @@ export function resolvePointPosition(chart, options) {
   }
   return getChartCircle(chart, options);
 }
+
+export function requireVersion(min, ver) {
+  const parts = ver.split('.');
+  if (!min.split('.').reduce((a, c, i) => a && c <= parts[i], true)) {
+    throw new Error(`Chart.js v${ver} is not supported. v${min} or newer is required.`);
+  }
+}
```

Comment:
Will go bad on major release.

I think there also should be an error for that the Chart.js version is too high because of chance of breaking changes.

Reply:
I disagree on the error for too high version, because we can't know it breaks. However, if turns it does, a patch release can be published with the error.

Reply:
Valid point, but then it still should check first if the major version is higher as the current because now it sees `4.0.0` as a lower version as `3.7`